scootapi/server/setup.go

```diff
@@ -92,3 +95,21 @@ func RunServer(bag *ice.MagicBag, schema jsonconfig.Schema, config []byte) {
 	}()
 	log.Fatal("Error serving: ", <-errCh)
 }
+
+// GetConfigText finds the right text for a configFlag.
+// If configFlag looks like a filename (of the form foo.bar where foo and bar are just alphanumeric),
+// read it as an asset.
+// Otherwise, assume it's the literal json text.
+func GetConfigText(configFlag string, asset func(string) ([]byte, error)) ([]byte, error) {
```

Comment:
So I disagree with having this method in the Server code.  I think it belongs as a private method in the binary, or perhaps as a util function in the config package, since workerserver and other binaries may want to use this convention.

Putting it here couples how we read and get input for scootapi to the scootapi implementation.  

This code also needs test cases